deployment/scripts/chainspec/dev_chainspec.json

```diff
@@ -1,10 +1,5 @@
 {
   "chain_name": "Testnet",
-  "block_production": {
```

Comment:
Why is deployment chainspec getting changed in a publishing fix?

Reply:
This section has been replaced by a "consensus" portion. We no longer use the chainspec for poa trigger configuration since verifiers can't check things like whether or not the poa produced the block instantly. Chainspec should only contain fields necessary for verification of blocks, not general node-specific parameters that can't be checked by anyone else.